docs/source/index.rst

```diff
@@ -49,9 +57,127 @@ a dialog into sub-dialogs and process them separately.
 Each `flow` represent a sub-dialog corresponding to the discussion of a particular topic.
 Each `flow` is also a dictionary, where the keys are the `nodes`.
 
-**Node**: Each `node` necessarily contains a `response` of the bot and
-the `condition` of `transition` to another `node`
-in this or another `flow`.
+**Node**: A `node` is the smallest unit of a dialog `flow`, and it contains the bot's response to a user's input as well as a `condition` that determines
+the `transition` to another `node`, whether it's within the current or another `flow`.
+
+ToCs
+----
+
+Context Storages
+~~~~~~~~~~~~~~~~
+
+- **Redis:** Provides a Redis-based version of the :py:class:`.DBContextStorage` class.
+
+- **Protocol:** This module contains base protocol code. Supported protocols fot db:
+   shelve, json, pickle, sqlite, redis, mongodb, mysql, postgresql, grpc, grpcs.
+
+- **SQL:** Provides a SQL-based version of the :py:class:`.DBContextStorage` class.
+   It allows the user to choose the backend option of his liking from MySQL, PostgreSQL, or SQLite.
+
+- **Mongo:** Provides a MongoDB-based version of the :py:class:`.DBContextStorage` class.
+
+- **JSON:** Provides a json-based version of the :py:class:`.DBContextStorage` class.
+
+- **Pickle:** Provides a pickle-based version of the :py:class:`.DBContextStorage` class.
+
+- **Database:** This module contains the Database class which is used to store and retrieve context data.
+
+- **Shelve:** Provides a shelve-based version of the :py:class:`.DBContextStorage` class.
+
+- **Yandex DB:** Provides a version of the :py:class:`.DBContextStorage` class that is specifically designed
+   to work with Yandex DataBase.
+
+
+Messenger Interfaces
+~~~~~~~~~~~~~~~~~~~~
+
+- **Message Interfaces:** This module contains several basic classes of message interfaces.
+   These classes provide a standardized way of interacting with different messaging services,
+   allowing the application to work with multiple messaging platforms seamlessly.
+
+- **dff.messengers.telegram package:** This package contains classes and functions specific
```

Comment:
`dff.messengers.telegram package` - i think, that it is bad naming and we can drop `package` word from name

Reply:
Also I recommend to change ToC to Table of Content

Reply:
I meant use instead `dff.messengers.telegram package` some thing like  "Telegram Interface"